distributed/worker.py

```diff
@@ -554,7 +554,7 @@ def __init__(
             self._workdir = self._workspace.new_work_dir(prefix="worker-")
             self.local_directory = self._workdir.dir_path
 
-        if preload is None:
+        if not preload:
             preload = dask.config.get("distributed.worker.preload")
         if preload_argv is None:
```

Comment:
It looks like `preload_argv` will also be an empty tuple if no value is used for the `--preload-argv` CLI option for `dask-worker`

```suggestion
        if not preload_argv:
```

Reply:
Study a lot from your reply :)
I just know it can insert code snippet  here.
Thank you~~